source/features/index.md

```diff
@@ -40,25 +40,22 @@ NodeX Agent enables devices to autonomously register the public key to the decen
 **Figure 7.** Comparison of conventional and automated provisioning flow
 :::
 
-## E2E Secure Channel
-
-Two-way communication (device-cloud, device- device) happens via a message handler, called by the customer, NodeX Hub. NodeX Agent stores NodeX Hub's DID in RoT SPE as pre-configuration. The device performs handshakes with the NodeX Hub to establish an end-to-end secure channel. In the handshake process, the system sends the public key (for encryption) and encrypted and signed message including client_id and client_secret to the server as client hello. The server generates a common key with a received public key, and decrypts the message to verify the device's signature and client_secret, and responds with an encrypted and signed message to the device. The device then verifies the server's signature and compares the server's DID with the pre-configured DID to authenticate the server, and completes the handshake process.
-
 ## Access Control
 
-The system software stores in PSA-RoT secure storage a security policy regarding access privileges. Such a security policy is used by the system software to manage access privileges. Device security parameters can only be updated by privileged users who are allowed to access the NodeX Hub. There are two access privilege levels; Admin and User. Admin and User (including cloud application through API) are authenticated and authorized by Hub. Admin users are allowed to send encrypted and signed data with secret_token to the target device, and the device validates the secret_token using a pre-configured security policy to authenticate the admin user. In the system software, the execution of application functionalities is enabled with the lowest level of privilege necessary for the intended function.
+The system software stores in PSA-RoT secure storage a security policy regarding access privileges. Such a security policy is used by the system software to manage access privileges. Device security parameters can only be updated by privileged users who are allowed to access the NodeX Studio. There are two access privilege levels; Admin and User. Admin and User (including cloud application through API) are authenticated and authorized by Studio. Admin users are allowed to send encrypted and signed data with secret_token to the target device, and the device validates the secret_token using a pre-configured security policy to authenticate the admin user. In the system software, the execution of application functionalities is enabled with the lowest level of privilege necessary for the intended function.
 
 ## Logging
```

Comment:
There are many instances in the "Log" section that require correction. So, it would be acceptable to remove the entire block for now.